examples/e2e_counter/reader.go

```diff
@@ -9,6 +9,7 @@ import (
 	ma "github.com/multiformats/go-multiaddr"
```

Comment:
A quick refresher: This `e2e_counter` was some of the first e2e things that I created when we had a first working version of _DB_. We have chatted some times to remove this code if may be confusing or adding too much burden to refactors or maintainability.  No strong opinion, but just refreshing this since I _originally_ made this _example_... no feelings with it. If it adds value in some direction, np with leaving this too.

The _books_ example was made roughly at the same time... but feels that example it's much nicer to show somebody a minimal way of using things.

Just leaving al this as a note here.

Reply:
Ah ok. If anyone has some strong feeling about removing it, I'm fine with that, but also don't have a strong opinion.